packages/api/src/resolvers.ts

```diff
@@ -36,12 +32,23 @@ const resolvers = {
       return (await resultRequestRepository.getLastResult(parent.feedFullName))
         ?.result
     },
+    lastResultTimestamp: async (
+      parent,
+      _args,
+      { resultRequestRepository }: Context
+    ) => {
+      return (await resultRequestRepository.getLastResult(parent.feedFullName))
```

Comment:
We are doing the same call in different resolvers so we are overfetching the db. The most common way to avoid this problem in graphql is adding a library such as [dataloader](https://www.npmjs.com/package/dataloader). IMO this is not a issue right now but we should add a comment and consider adding it if we add more fields that make the same call.
```suggestion
      // FIXME: add dataloader library to avoid overfetching
      return (await resultRequestRepository.getLastResult(parent.feedFullName))
```